image_to_table/server/app/routes.py

```diff
@@ -0,0 +1,14 @@
+from http import HTTPStatus
+
+from flask import Flask, make_response, request
+
+
+def api():
+    x = request.json
+    y = request.files
+    z = request.data
```

Comment:
`z` is the data from the request. See the example request in the txt file.

Reply:
So I'll clean this up a bit before merging. Just wanted to find out how to take an image directly as input

Reply:
Japp, maybe better names than x, y and z? 😄

Reply:
😁